src/components/IntegrityBlockPane.vue

```diff
@@ -31,23 +38,40 @@ function stringifySignature(signature: Uint8Array) {
 }
 
 const data = computed(() => {
-  const block = props.bundle.signatureBlock!;
+  const { attributes, signatures } = props.bundle.integrityBlock!;
   return {
-    'Computed Origin': computeOrigin(block.ed25519PublicKey),
-    'Public Key': stringifyKey(block.ed25519PublicKey),
-    Signature: stringifySignature(block.signature),
-  };
+    computedOrigin: attributes
+      ? `isolated-app://${attributes.webBundleId}`
+      : computeOrigin(signatures[0].publicKey, signatures[0].type),
+    signatures: signatures.map((signature) => {
+      return {
+        Type: SignatureType[signature.type],
+        'Public Key': stringifyKey(signature.publicKey),
+        Signature: stringifySignature(signature.signature),
+      };
+    })
+  }
 });
 </script>
 
 <template>
   <Header sticky>Integrity Block</Header>
   <div class="px-1">
     <dl class="inline-grid grid-cols-[auto_1fr] gap-x-3">
-      <template v-for="(value, key) in data">
-        <dt class="text-secondary">{{ key }}</dt>
-        <dd class="font-mono">{{ value }}</dd>
-      </template>
+      <dt class="text-secondary">{{ 'Computed Origin' }}</dt>
```

Comment:
This is how the page looks with this code:

![Screenshot from 2024-07-10 12-57-04](https://github.com/robbiemc/wbn-explorer/assets/29898901/cd9f70cf-2693-40ea-ab52-8fa124907a16)

The text is somewhat misaligned (I have to admit that I'm really bad when it comes to markup).

Reply:
For now I'd submit this as-is. We can try CSS Grid or something like that to get the alignment sorted out in a followup.